tests/ResourceTest.php

```diff
@@ -301,4 +303,45 @@ public function revisions_cant_be_enabled_without_publish_states()
 
         $this->assertFalse($resource->revisionsEnabled());
     }
+
+    #[Test]
+    public function scope_runway_search_works_with_custom_eloquent_connection()
+    {
+        Config::set('database.connections.external', [
+            'driver' => 'sqlite',
+            'database' => ':memory:',
+            'prefix' => '',
+        ]);
+
+        Schema::connection('external')->create('external_posts', function ($table) {
+            $table->id();
+            $table->string('title');
+            $table->longText('body');
+            $table->timestamps();
+        });
+
+        Config::set('runway.resources.'.ExternalPost::class, []);
+
+        Runway::discoverResources();
```

Comment:
Let's call `registerResource` here instead. 

It avoids us needing to set the config, then tell Runway to find all the resources again. It's a tiny bit cleaner IMO.

```suggestion
        Runway::registerResource(ExternalPost::class, []);
```